Dockerfile

```diff
@@ -88,12 +88,15 @@ ADD . /go/src/github.com/mattermost/platform
 ADD ./docker/main.cf /etc/postfix/
 
 RUN go get github.com/tools/godep
-RUN cd /go/src/github.com/mattermost/platform; godep restore 
+RUN cd /go/src/github.com/mattermost/platform; godep restore
 RUN go install github.com/mattermost/platform
-RUN cd /go/src/github.com/mattermost/platform/web/react; npm install 
+RUN cd /go/src/github.com/mattermost/platform/web/react; npm install
 
 RUN chmod +x /go/src/github.com/mattermost/platform/docker/docker-entry.sh
 ENTRYPOINT /go/src/github.com/mattermost/platform/docker/docker-entry.sh
 
+# Create default storage directory
+RUN mkdir /mattermost/
```

Comment:
The Dockerfile doesn't actually need this line but I added to be explicit. The server will automatically create the directories it needs to save files